components/features/marketing/UseCaseExplorer.tsx

```
@@ -0,0 +1,248 @@
+                        </p>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </Card>
+            </div>
+          </div>
+        </div>
+      </div>
+      
+      <style jsx>{`
```

ESLint

[error] Unknown property 'jsx' found

(react/no-unknown-property)